vnext/Scripts/rnw-dependencies.ps1

```diff
@@ -38,7 +38,12 @@ function InstallVS {
     $vsWhere = "$installerPath\vswhere.exe"
     if (!(Test-Path $vsWhere)) {
         # No VSWhere / VS_Installer
-        & choco install -y visualstudio2019community
+        if ($Enterprise) {
+            # The CI machines need the enterprise version of VS as that is what is hardcoded in all the scripts
```

Comment:
we don't get here in CI because we find vswhere and all the payloads

Reply:
This is for setting up private VM agents, the hosted ones have enterprise already, but it is too painful to use those scripts to create images, so I'm trying to create images using this script. So far it is going good, besides the user issue running the UWP tests as discussed on teams.

Reply:
The private VM's are a hope to fix the OOM errors we are seeing for older branches and recently on the WinUI3CS validations.